src/main/flight/mixer.c

```diff
@@ -500,7 +500,7 @@ void mixTable(void)
     // Find min and max throttle based on condition.
     if (feature(FEATURE_3D)) {
         if (!ARMING_FLAG(ARMED)) throttlePrevious = rxConfig()->midrc; // When disarmed set to mid_rc. It always results in positive direction after arming.
-
+        rcCommand[THROTTLE] = rcCommand[THROTTLE] + 57; //Adjust Center Stick Accuracy, Issue #1879
```

Comment:
I second @DzikuVx's comment. A magic number that's only valid for certain configuration is not a good way to fix things.